### PR TITLE
LaTeX compliance for formulas and errors in bibliography

### PR DESCRIPTION
**Pull request by albert-github**
Correction of incorrect usage of LaTeX  in formulas and bibliography altough MatHJax and bibtex.pl doesn't always signal it.
- Incorrect biblio entry (missing `,` and `}`)
  - Documentation/doc/biblio/geom.bib
- `&nbsp` is not correct LaTeX has to be `~`
  - Arrangement_on_surface_2/doc/Arrangement_on_surface_2/Arrangement_on_surface_2.txt
  - Documentation/doc/Documentation/Developer_manual/Chapter_intro.txt
- Incorrect formula regarding usage of `\left` and `\right` (also signaled by MathJax
  - Arrangement_on_surface_2/doc/Arrangement_on_surface_2/Arrangement_on_surface_2.txt
- `DeclareMathOperator` is a command that should be in the preamble, alternatively the command `\operatorname` can be used explicitly (as done here as it is only used once)
  - Kinetic_surface_reconstruction/doc/Kinetic_surface_reconstruction/Kinetic_surface_reconstruction.txt
- Latex has problems with `_` in a `text...` command so it should be escaped though this gives problems with MathJax hence the extra hook. MathJax reference: https://groups.google.com/g/mathjax-users/c/wSh6-hSIUpQ/m/KmzZhQQGslgJ
  - Documentation/doc/resources/1.10.0/CGAL_mathjax.js
  - Documentation/doc/resources/1.8.13/CGAL_mathjax.js
  - Documentation/doc/resources/1.9.6/CGAL_mathjax.js
  - Weights/include/CGAL/Weights/authalic_weights.h

### CODE REVIEW

**Review by afabri — APPROVED** [review]

**Comment by afabri** [comment]
I guess it would be trivial to apply that to the 6.0 branch

**Comment by albert-github** [comment]
Why no label "Tested" and remove "Under Testing"?
